### ISSUE TITLE
Bug: One page previews of office documents are blank

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#3401]
## Description and expectations

Downloading the first PDF preview page of a office document leads to a blank page. This bug exists on all libreoffice mimetypes and maybe on other formats too.

Preview of *.doc document not working, no error visible in tracim but visible in backend about lock (see section Relevant Logs).

### How to reproduce

- Open *.doc document
- download first page in pdf for a libreoffice document

### Version information

- Tracim 3.0.0
- Preview generator 0.14

### Extra information

Related preview-generator issue : algoo/preview-generator#194

### Original issue:

timeout issue part   (which is another issue)  has been moved to https://github.com/tracim/tracim/issues/3459

### Implemented solution

https://github.com/tracim/tracim/pull/3457

- update preview generator from 0.14 to 0.15.4 which solve this bug, see https://github.com/algoo/preview-generator/pull/200

___
Edited 2020-08-26: Split the 2 bugs explained in this issue into 2 different issue. Add implemented solution for this one.

**Comment by inkhey** [comment]
After some research, i discovered the issue is related to the fact we do use stream object instead of path since release 0.12 for pdf decryption:

For unclear reason, pdf decryption using stream object doesn't work as expected in this specific case.

see :+1:

```
I don't see how this worked before, as `get_decrypted_pdf` calls `seek(0)` on its first parameter...

_Originally posted by @JulienPalard in https://github.com/algoo/preview-generator/pull/100#discussion_r303020089_
```

**Comment by PhilippeAccorsi** [comment]
Preview-generator release 0.12 is introduced in tracim release_02.06.00

**Comment by raphj** [comment]
@PhilippeAccorsi 

After discussion with @grignards and @inkhey,

**Comment by gbonaspetti** [comment]
Tested on branch.

**Comment by PhilippeAccorsi** [comment]
Available in 3.0.4
